src/ragapp/backend/models/tools/image_generator.py

```diff
@@ -6,21 +6,22 @@
 class ImageGeneratorToolConfig(BaseModel):
     api_key: str | None = Field(
         default=None,
-        description="The Stability AI API key to use for the image generator.",
+        description="The API key to use for the Image Generator.",
```

Comment:
if we don't show that string in the FE, then it's not an issue, but this is the Stability AI API key